apps/api/src/app/integrations/integrations.controller.ts

```diff
@@ -173,6 +175,7 @@ export class IntegrationsController {
   }
 
   @Put('/:integrationId')
+  @UseGuards(RolesGuard)
```

Comment:
are we sure we want to add this here? I would actually remove the @Roles decorator instead, given that we didn't check roles up until now

Reply:
I agree, I wanted to keep the original implementation that was there before but then it would be more than defined in DoD - will remove